core/jobs/src/main/java/com/cognifide/aet/job/common/modifiers/js/JsModifier.java

```diff
@@ -0,0 +1,88 @@
+/**
```

Comment:
Please add a documentation for this modifier. The instruction how to do this can be found in our [contributing rules](https://github.com/Cognifide/aet/blob/master/CONTRIBUTING.md#documentation).
You should create a new page in `documentation` module.